### PR TITLE
introduction to Galaxy edits

### PR DESCRIPTION
**Pull request by bagnacan**
- edited the content of a few slides
- edited the comment section (presentation mode) of a few slides

### CODE REVIEW

File: topics/introduction/slides/index.html [hunk 1]
```diff
@@ -37,8 +37,14 @@
 
 ???
 
-- Information captured by Galaxy include: tool version, inputs and parameters.
-- Pages: interactive, web-based documents that describe a complete analysis.
```
Comment (nsoranzo) [inline topics/introduction/slides/index.html]: Any specific reason to remove the note about Pages? Can you restore it?

File: topics/introduction/slides/index.html [hunk 2]
```diff
@@ -93,30 +99,35 @@
...
-  - Hidden, deleted, purged
+  - name, format, size, creation time, datatype-specific metadata
+  - tool id and version, inputs, parameters
+  - standard output (`stdout`) and error (`stderr`)
+  - state (<span style="background-color: grey">waiting</span>, <span style="background-color: yellow">running</span>, <span style="background-color: green">success</span>, <span style="background-color: red">failed</span>)
+  - hidden, deleted, purged
 
 ???
 
-Purged means permanently deleted
```
Comment (nsoranzo) [inline topics/introduction/slides/index.html]: Restore this line?

File: topics/introduction/slides/index.html [hunk 3]
```diff
@@ -93,30 +99,35 @@
 - how to generate a command to execute the tool with the specified inputs and parameters
 - the output datasets the tool should produce and their datatypes
 
+Tools can be viewed as tiny LEGO pieces: each one solves a specific problem, and each can be combined to build complex data analysis.
```
Comment (nsoranzo) [inline topics/introduction/slides/index.html]: "each can be combined to build complex data analysis" -> "they can be combined together to build complex analysis pipelines"

File: topics/introduction/slides/index.html (hunk 1 above)
Comment (bagnacan) [reply topics/introduction/slides/index.html]: Sorry, I deleted that by mistake. Thanks @nsoranzo!

File: topics/introduction/slides/index.html [hunk 4]
```diff
@@ -37,8 +37,14 @@
 
 ???
 
-- Information captured by Galaxy include: tool version, inputs and parameters.
-- Pages: interactive, web-based documents that describe a complete analysis.
+**accessible** **reproducible** **transparent** research means <i>sharing everything</i>.
+
+If the Galaxy framework makes everything as simple as possible, researchers are able to:
+- share orkflow analysis with their colleagues
```
[the remark this answers is not in the document]
Comment (bagnacan) [reply topics/introduction/slides/index.html]: more concise and correct, thanks!

File: topics/introduction/slides/index.html [hunk 5]
```diff
@@ -37,8 +37,14 @@
...
+- track all used tools and versions
+- check each parametrizations
```
[the remark this answers is not in the document]
Comment (bagnacan) [reply topics/introduction/slides/index.html]: better. I edited accordingly

File: topics/introduction/slides/index.html (hunk 3 above)
Comment (bagnacan) [reply topics/introduction/slides/index.html]: clearer, thanks

File: topics/introduction/slides/index.html (hunk 2 above)
Comment (bagnacan) [reply topics/introduction/slides/index.html]: done

File: topics/introduction/slides/index.html [hunk 6]
```diff
@@ -35,10 +35,18 @@
...
-- Information captured by Galaxy include: tool version, inputs and parameters.
-- Pages: interactive, web-based documents that describe a complete analysis.
+**accessible** **reproducible** **transparent** research means <i>sharing everything</i>.
+
+If the Galaxy framework makes everything as simple as possible, researchers are able to:
+- share their analyses
+- track all used tools and versions
+- check all parameters
+- justify each step in the analysis
+- publish the findings with all aforementioned informations
```
Comment (nsoranzo) [inline topics/introduction/slides/index.html]: s/informations/information/ , it has no plural in English.

Comment (bagnacan) [reply topics/introduction/slides/index.html]: thanks, @nsoranzo!

**Comment by nsoranzo** [comment]
Thanks @bagnacan!

**Pull request merged**